src/Libraries/Microsoft.Extensions.Resilience/README.md

```diff
@@ -18,6 +18,22 @@ Or directly in the C# project file:
 </ItemGroup>
 ```
 
+## Usage Examples
+
+The services can be registered using the following method:
+
+```csharp
+public static IServiceCollection AddResilienceEnricher(this IServiceCollection services)
```

Comment:
I put a semi-colon on these API blocks. We should standardize when all the READMEs are in

Reply:
How does the semi-colon help? Neither are syntactically correct. This is a method definition which would be followed by `{ ... }`. A semi-colon would make sense if this was a calling example like `serviceCollection.AddResilienceEnricher()`.